projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.ts

```diff
@@ -77,6 +79,11 @@ export class ListViewComponent implements OnInit {
         this._doSearch();
     }
 
+    ngOnChanges(): void {
```

Comment:
Lifecycle interface OnChanges should be implemented for method ngOnChanges.

Reply:
Done in f0268ac